moto/route53/models.py

```diff
@@ -64,10 +64,26 @@ def __init__(self, health_check_id, caller_reference, health_check_args):
         self.measure_latency = health_check_args.get("measure_latency") or False
         self.inverted = health_check_args.get("inverted") or False
         self.disabled = health_check_args.get("disabled") or False
-        self.enable_sni = health_check_args.get("enable_sni") or False
-        self.children = health_check_args.get("children") or None
+        self.enable_sni = health_check_args.get("enable_sni") or True
         self.caller_reference = caller_reference
 
+    def add_children(self, children):
```

Comment:
The name `set_children` would be more accurate, as it doesn't add anything to the existing list of children

Reply:
Agreed. Updated